libsel4/tools/__init__.py

```diff
@@ -8,4 +8,6 @@
 # This file allows the the libsel4 tools directory to be imported as
 # a python module.
 
+from . import loader
```

Comment:
Why is the new loader module needed?

Reply:
I added a large comment explaining it

Reply:
In `syscall_stub_gen.py`, can't  we do 
```
sys.path.append(str(pathlib.Path(os.path.dirname(os.path.realpath(__file__))).parent))
```
to fix this?

Reply:
Presumably, that would work but I think it would still run into the autopep8 reordering problem.
We would also need to get rid of the `.parent` stuff.

Reply:
Can you create an issue for this, then it's kept track of?

Reply:
Sure, but prefer to do it after merge, so we don't end up with dangling issues.

Reply:
I filed #811 for this now.